selfdrive/car/hyundai/values.py

```diff
@@ -14,9 +14,6 @@
 
 
 class CarControllerParams:
-  ACCEL_MIN = -3.5 # m/s
-  ACCEL_MAX = 2.0 # m/s
```

Comment:
This vary per brand (sometimes), and should probably remain inside the port. However, we can maybe make a common or base CarControllerParams to clean up some of the duplication

Reply:
ok.. brand value revert...